.coveragerc

```diff
@@ -0,0 +1,15 @@
+[run]
+omit =
+    # graph.py
+    pym/bob/cmds/graph.py
```

Comment:
I think we could add a simple black-box-test that generates a dot and d3 graph based on some simple recipes. That  doesn't need to be complete but only a smoke test that shows that the code at least still runs. Whether it generates something sensible cannot be tested, that's clear.